README.md

```diff
@@ -130,6 +130,9 @@ To keep all those hundreds of source files from ending up in the same directory,
 
 First, create them as groups (little yellow "folders") within the group with your project's name in Xcode's Project Navigator. Then, for each of the groups, link them to an actual directory in your project path by opening their File Inspector on the right, hitting the little gray folder icon, and creating a new subfolder with the name of the group in your project directory.
 
+However, in case of a complex project, splitting the files based on the features is a good idea to keep in mind. 
+Some instances of the components mentioned before usually shared with most of the code of the project, but the main part of them are only related to a particular feature. So, makes sense to organise them under the same folder named with the specific feature.
```

Comment:
... are usualley shared...
..., but most of them ....

Also would it make sense to mention the new behaviour in Xcode9 where groups are stronger coupled with the filesystem/folders. So creating a group now creates also a folder, so the paragraph in line 131 is kinda out of date?

Reply:
Yes, I think this advice can be deleted now, that Xcode does the right thing. Maybe it still makes sense to point out that older code bases might still need to be updated.

Reply:
I agree that we can delete this now.